app/presenters/V1/nsm/disbursement_costs_presenter.rb

```diff
@@ -5,11 +5,12 @@ def initialize(disbursement)
         @disbursement = disbursement
       end
 
+      # TODO: Add real assessed costs if using caseworker assessed data
       def data_for_calculation
         {
           disbursement_type: @disbursement["disbursement_type"],
           claimed_cost: @disbursement["total_cost_without_vat"],
-          claimed_miles: BigDecimal(@disbursement["miles"].to_s),
+          claimed_miles: BigDecimal(disbursement_miles),
```

Comment:
`BigDecimal(@disbursement["miles"] || 0)`

Reply:
this fails as empty strings are considered truthy in ruby

Reply:
`.to_i` then